tensorflow_probability/examples/jupyter_notebooks/Fitting_DPMM_with_Symmetric_Dirichlet_Prior_Using_pSGLD.ipynb

```diff
@@ -0,0 +1,975 @@
+        "##### Copyright 2018 The TensorFlow Authors.\n",
+        "\n",
+        "Licensed under the Apache License, Version 2.0 (the \"License\");\n"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "JC-lgcjL3892",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "#@title Licensed under the Apache License, Version 2.0 (the \"License\"); { display-mode: \"form\" }\n",
+        "# you may not use this file except in compliance with the License.\n",
+        "# You may obtain a copy of the License at\n",
+        "#\n",
+        "# https://www.apache.org/licenses/LICENSE-2.0\n",
+        "#\n",
+        "# Unless required by applicable law or agreed to in writing, software\n",
+        "# distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+        "# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+        "# See the License for the specific language governing permissions and\n",
+        "# limitations under the License."
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "Um6EsGjz4RON",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "# Fitting Dirichlet Process Mixture Model with Symmetric Dirichlet Prior Using Preconditioned Stochastic Gradient Langevin Dynamics"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "vsGdTAPr2FtM",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "In this notebook, we will demonstrate how to cluster a large number of samples and infer the number of clusters simultaneously by fitting Dirichlet Process Mixture of Gaussian distribution with Symmetric Dirichlet Prior using Preconditioned Stochastic Gradient Langevin Dynamics (pSGLD). "
+      ]
+    },
+    {
+      "metadata": {
+        "id": "A7RysXxCBNS_",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "## Samples"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "k-aVH4b4Bxzo",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "First, we set up a toy dataset. We generate 50,000 random samples from three bivariate Gaussian distributions."
+      ]
+    },
+    {
+      "metadata": {
+        "id": "TNGwb4TdrlJO",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "from __future__ import absolute_import\n",
+        "from __future__ import division\n",
+        "from __future__ import print_function\n",
+        "\n",
+        "import tensorflow as tf\n",
+        "import tensorflow_probability as tfp\n",
+        "import numpy as np\n",
+        "import matplotlib.pyplot as plt; plt.style.use('ggplot')\n",
+        "import matplotlib.cm as cm\n",
+        "from tqdm import tqdm\n",
```

Comment:
I don't know if we'll guarantee tqdm being installed, so I might leave this out.

Reply:
Removed tqdm.